packages/web/server/queries/complex/pools/market.ts

```diff
@@ -15,16 +14,16 @@ export type PoolMarketMetrics = Partial<{
   feesSpent7dUsd: PricePretty;
 }>;
 
-const metricPoolsCache = new LRUCache<string, CacheEntry>(DEFAULT_LRU_OPTIONS);
+const metricPoolsCache = new RemoteCache();
 /** Get a cached Map with pool IDs mapped to market metrics for that pool. */
 export function getCachedPoolMarketMetricsMap(): Promise<
   Map<string, PoolMarketMetrics>
 > {
   return cachified({
     cache: metricPoolsCache,
     key: "pools-metrics-map",
-    ttl: 1000 * 10, // 10 seconds
-    staleWhileRevalidate: 1000 * 25, // 25 seconds, edge function run time limit
+    ttl: 1000, // 1 min
+    staleWhileRevalidate: 1000 * 60, // 1 hour
```

Comment:
The 1 min ttl makes sense. What is the reason for choosing such a high number for stale while revalidate? Wouldn't it be more than we need?

Reply:
FYI I just fixed these values

What stale time are you thinking.

Reply:
I would expect 2 minutes at most to be useful and unsure why 1 hour is necessary.

From my understanding, after 1 minute we would update the cache in the background and `staleWhileRevalidate` represents how long the cache is still valid during the update.

The update workload will probably not take this long so I think we should lower it to be near of what we would expect

Reply:
perhaps 30 min?

Reply:
I'm down for 30 mins, as volumes quite a bit with market movement / hype

Reply:
`staleWhileRevalidate`:
```
 * Amount of milliseconds that a value with exceeded ttl is still returned
     * while a fresh value is refreshed in the background
```

I think it makes sense to have this be at most 1-2 minutes. Do we expect the refresh workloads to run longer than that?

Reply:
If we're more generous with the stale time, then offers more tolerance to the service being down for 1 min as you suggest

Reply:
I see the goal you're trying to achieve with this now